types/relay-compiler/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for relay-compiler 8.0
+// Type definitions for relay-compiler 14.0
 // Project: https://relay.dev
 // Definitions by: n1ru4l <https://github.com/n1ru4l>
 //                 Eloy Durán <https://github.com/alloy>
```

Comment:
Looks like we lost the added lines here
```suggestion
//                 Eloy Durán <https://github.com/alloy>
//                 Marais Rossouw <https://github.com/maraisr>
//                 Levi Buzolic <https://github.com/levibuzolic>
```